content/04-class-field.md

```diff
@@ -333,7 +333,16 @@ Main.main();
 })({});
 ```
 
+Note that even though we call such kind of fields "variables", inline variables can never
+be reassigned as the value must be known at compile-time to be inlined at the place of usage.
+This makes inline variables a subset of [`final` fields](class-field-final), hence the usage
+of the `final` keyword in the code example above.
 
+> ##### Trivia: `inline var`
+>
+> Prior to Haxe 4, there was no `final` keyword, but inline variables feature was present
```

Comment:
I think this would be better in two sentences – "Prior to … keyword. Inline variables …"

Also, "Inline variables were present …" or "The inline variables feature was present …"

Reply:
Well the point is to say that even though there was no "final" keyword, inline vars were already there.

Reply:
I understand the intent. Maybe connect the sentences with a `–` rather than `, but`?